packages/react/src/hooks/actions/shared/useTypeCastFields.ts

```diff
@@ -1,16 +1,18 @@
 import * as React from 'react';
 import {
   ModelInit,
+  PersistentModel,
   PersistentModelMetaData,
   Schema,
 } from '@aws-amplify/datastore';
 
 import { DataStoreActionFields, isAlreadyTyped } from './types';
+import { isString } from 'lodash';
```

Comment:
For proper treeshaking, I think this is supposed to be:

```suggestion
import { isString } from 'lodash/isString';
```

Reply:
I believe this should be default import iirc